packages/config/config/devices/0x027a/zen23.json

```diff
@@ -28,6 +32,28 @@
 	"paramInformation": [
 		{
 			"#": "1",
+			"$if": "firmwareVersion >= 1.5 && firmwareVersion < 3.0 || firmwareVersion === 20.15",
+			"label": "Switch behavior",
+			"description": "Determines whether up is on and down is off; or up is off, down is on",
```

Comment:
This seems unnecessary, the option labels meintion this already.

Reply:
I removed the description since as you said it's redundant. It was a copy/paste of the existing config, I had not intended to any review of the labels while adding these conditional parameters. 

I'm sure you're aware but this parameter label (and others) are inconsistent throughout the Zooz files. It sometimes called 'Toggle control' or 'Orientation' also with different labels for the values. From what I've seen it's probably all taken from how Zooz has changed the language over time, different descriptors each product generation.

I notice from your style guide that you're big on consistency-- and are using templates for config files! So I had assumed there was probably a larger scope cleanup of the Zooz files planned for the future.

Reply:
I also added the spaces, thank you!

Reply:
> there was probably a larger scope cleanup of the Zooz files planned for the future

There is, but we still try to clean up the files a bit if they are touched anyways.